src/model/data/fridge/REST.yaml

```diff
@@ -459,8 +512,8 @@ components:
           type: string
           enum: [good, dirty, out of order, not at location, ghost]
 
-        foodPercentage:
-          description: The percentage of food in the fridge.
+        foodAmount:
```

Comment:
foodAmount is more fitting, but this will require a data migration so I would keep it as foodPercentage for now

Reply:
But will create a ticket for it: https://trello.com/c/DIpAHBPK/249-cfm220-change-foodpercentage-to-foodamount